CI/macos/appcast_convert.py

```diff
@@ -0,0 +1,83 @@
+import os
+
+import xmltodict
+
+
+DELTA_BASE_URL = 'https://cdn-fastly.obsproject.com/downloads/sparkle_deltas'
+
+
+def convert_appcast(filename):
```

Comment:
Run this file through `black` without any specific options.

Reply:
Black actually does nothing to this file except changing the quote from `'` to `"` (although my default is to run it with that disabled).

Reply:
Ran it through black with string normalisation enabled, as I said the only thing changed was the quotes.